lms/djangoapps/support/static/support/jsx/entitlements/data/reducers/form.js

```diff
@@ -0,0 +1,24 @@
+import { formActions, entitlementActions } from '../actions/constants';
+
+const clearFormState = {
+  type: '',
+  isOpen: false,
+  activeEntitlement: null,
+};
+
+const form = (state = {}, action) => {
+  switch (action.type) {
+    case formActions.OPEN_REISSUE_FORM:
+      return { ...state, type: 'reissue', isOpen: true, activeEntitlement: action.entitlement };
```

Comment:
`"reissue"` and `"create"` are used in a couple different places - it might be useful, from a maintenance perspective to put them in a constant so if you ever have to change it to `REISSUE`